src/analysis/Observable.hpp

```diff
@@ -69,7 +69,7 @@ class Observable : public SystemAccess
     /** returns observable of type int, used for Python and on C++ level*/
     virtual int compute_int() const { return 0; };
     /** computes vector of real values (e.g. pressure tensor, ...), used on C++ level only */
-    virtual void compute_real_vector() { return; };
+    virtual std::vector<real> compute_real_vector() { return std::vector<real>(); };
```

Comment:
Is throwing a not implemented error a better default?

Reply:
I would keep it as it is - other virtual method of this class returns already default values. I'm not sure what will break if I throw an exception here.